src/page/auth.ejs

```diff
@@ -26,15 +26,22 @@
     <script src='../libs/countly/countly.min.js?v=<%= VERSION %>' type='text/javascript'></script>
     <script src='../min/countlyBoomerangCustom.js?v=<%= VERSION %>' type='text/javascript'></script>
     <script type='text/javascript' nonce="<%= COUNTLY_NONCE %>">
-      Countly.init({
-        debug: false,
-        app_key:"<%= COUNTLY_API_KEY %>",
-        url: "https://countly.wire.com/",
-        offline_mode: true,
-        use_session_cookie: false,
-        storage: 'localstorage'
-      });
-      Countly.track_performance();
```

Comment:
can you explain why this change? just curious

Reply:
While parsing through the documentation I saw that they recommend the asynchronous implementation with the q - array. The synchronous one is optional but inferior (resource blocking).
Since we can not use the SDK directly anymore, because of the performance plugin integration, our use case for a synchronous implementation is out of the window. 
Therefore it is a good call to change how we use countly to the recommended way.